packages/react/src/common/useFocusWithArrows.tsx

```diff
@@ -0,0 +1,107 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import type { KeyboardEvent, RefObject } from 'react'
+
+const KeyboardKeys = {
+  ArrowUp: 'ArrowUp',
+  ArrowDown: 'ArrowDown',
+  ArrowRight: 'ArrowRight',
+  ArrowLeft: 'ArrowLeft',
+  Home: 'Home',
+  End: 'End',
+}
+
+const FOCUSABLE_ELEMENTS = [
+  'a[href]:not([disabled])',
+  'button:not([disabled])',
+  'textarea:not([disabled])',
+  'input[type="text"]:not([disabled])',
+  'input[type="radio"]:not([disabled])',
+  'input[type="checkbox"]:not([disabled])',
+  'select:not([disabled])',
+]
+
+const useFocusWithArrows = (
+  ref: RefObject<HTMLDivElement>,
+  rotating = false,
+  directChildrenOnly = false,
+  horizontally = false,
```

Comment:
Can we group these into an `options` object? This pattern has various advantages, e.g. see [this SO conversation](https://stackoverflow.com/questions/12826977/multiple-arguments-vs-options-object) and [this article](https://medium.com/quick-code/maintainable-options-parameters-for-functions-in-javascript-9f7d5d207a65).